explainaboard/metrics/nlg_meta_evaluation.py

```diff
@@ -188,10 +188,21 @@ def _count(self, score: list, config: Optional[MetricConfig] = None):
         num = 0
         for i in range(1, len(score)):
             for j in range(0, i):
-                if abs(score[i][0] - score[j][0]) >= config.threshold:
-                    manual_better = score[i][0] > score[j][0]
-                    system_better = score[i][1] > score[j][1]
-                    if manual_better == system_better:
+                if (
+                    abs(score[i][0] - score[j][0]) < config.threshold
+                    or abs(score[i][0] - score[j][0]) == 0
+                ):
+                    continue
+                elif (
+                    score[i][0] - score[j][0] >= config.threshold
+                ):  # system i is better than system j
+                    if score[i][1] > score[j][1]:
+                        conc += 1
+                    else:
+                        disc += 1
+                    num += 1
+                else:  # system i is worse than system j
+                    if score[i][1] < score[j][1]:
```

Comment:
It looks the same stats are used multiple times. Could you apply following variables to judge the conditions?

- `manual_diff = score[i][0] - score[j][0]`
- `system_diff = score[i][1] - score[j][1]`

Reply:
good suggestion